chart/kepler/templates/secret-redfish.yaml

```
@@ -0,0 +1,22 @@
+{{- if .Values.redfish.enabled }}
+{{- with .Values.redfish }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    {{- include "kepler.labels" $ | nindent 4 }}
+    {{- with .labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- with .annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+type: Opaque
+stringData:
+  redfish.csv: {{ .fileContent }}
```

It works fine when this point is fixed:
```suggestion
  redfish.csv: {{- .fileContent | toYaml | indent 2 }}
```

---

When you say it works, are you referring to the deployment is successful, the application is able to read that file and do what ever redfish should do?

---

Yes, Kepler was deployed in success, then we can see metrics from Redfish on Kepler exporter.